frontend/package.json

```diff
@@ -107,7 +108,8 @@
     "stylelint": "^13.13.1",
     "stylelint-config-recommended": "^3.0.0",
     "stylelint-config-standard": "^20.0.0",
-    "typescript": "^4.9.5"
+    "typescript": "^4.9.5",
+    "url-loader": "^4.1.1"
```

Comment:
just curious, where is this used? Thank you!

Reply:
I adapted this from sci-components. When removing url-loader, the svgs don't load properly as react components

Reply:
I see! Thanks for the context 🙏 Maybe I'll just take a mental note to look into this in the future. Thanks again for adding it!